Contribute/dotnet/dotnet-contribute.md

```diff
@@ -77,6 +81,23 @@ Be sure to follow the proper Markdown syntax. For examples of common , see the [
           /media
             /porting-overview
                 portability_report.png
+          /snippets
+            /csharp
+              porting.csproj
+              porting-overview.cs
+              Program.cs
+            /fsharp
+              porting.fsproj
+              porting-overview.fs
+              Program.fs
+            /vb
+              porting.vbproj
+              porting-overview.vb
+              Program.vb
+
+The structure shown above includes one image, *portability_report.png*, and three code projects that include **code snippets** that are included in the *porting-overview.md* article.
```

Comment:
I think this should show more of the standard structure. This is assuming that this article is the only one in the folder.

Reply:
That doesn't look right.  Should be:

```
/snippets
  /porting-overview
    /csharp
       porting.csproj
        porting-overview.cs
        Program.cs
    /fsharp
      porting.fsproj
      porting-overview.fs
      Program.fs
    /vb
      porting.vbproj
      porting-overview.vb
      Program.vb
```

Reply:
>  Do you know who needs to approve this before merging? Or was there a issue with recording your review?

When it's ready to go, add the `#sign-off` comment and tag Jim Parker.

Reply:
I thought we'd agreed on the structure @tdykstra added in his comment. I'll make that update.